src/Resolvers/Microsoft.DotNet.NativeWrapper/NETCoreSdkResolverNativeWrapper.cs

```diff
@@ -50,5 +50,32 @@ public static string[] GetAvailableSdks(string dotnetExeDirectory)
 
             return list.Entries;
         }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct hostfxr_initialize_parameters
+        {
+
+        }
+
+        public static int InitializeForRuntimeConfig(string runtimeConfigPath)
+        {
+            var result = -1;
+            IntPtr hostContextHandle = default;
+
+            hostfxr_initialize_parameters parameters = new hostfxr_initialize_parameters();
+
+            IntPtr parametersPtr = Marshal.AllocHGlobal(Marshal.SizeOf(parameters));
+            Marshal.StructureToPtr(parameters, parametersPtr, false);
```

Comment:
The parameters are not used for anything... do we need this?

Reply:
This still needs to be removed.